README.rst

```diff
@@ -20,6 +20,13 @@
 Qbsolv
 ======
 
+**NOTICE**: This repository is deprecated. Support will be discontinued in 2022.
+Please update your code to use Ocean's
```

Comment:
Why not specify march like in https://github.com/dwavesystems/dwave-ocean-sdk/pull/173?

Reply:
Oh yeah, I wrote a much better message there.